src/gui/screen_printing_serial.cpp

```diff
@@ -13,6 +13,35 @@
 #pragma pack(push)
 #pragma pack(1)
 
+typedef enum {
```

Comment:
please use enum class in C++ ... avoid old C-style constructs

Reply:
and why do you need `#pragma pack(1, push)` just for the enum? The enum doesn't care about alignment ...

Reply:
added C++ enum definition